src/shared/treeview/utils.ts

```diff
@@ -77,7 +77,7 @@ export function createPlaceholderItem(message: string): TreeNode {
     return {
         id: 'placeholder',
         resource: message,
-        treeItem: new vscode.TreeItem(message, vscode.TreeItemCollapsibleState.None),
+        getTreeItem: () => new vscode.TreeItem(message, vscode.TreeItemCollapsibleState.None),
```

Comment:
I don't like turning otherwise constant things into functions but it was kind of necessary for this functionality.
It makes things harder to test and reason about :/